e2e/etcd_release_upgrade_test.go

```diff
@@ -37,7 +35,11 @@ func TestReleaseUpgrade(t *testing.T) {
 	copiedCfg := configNoTLS
 	copiedCfg.execPath = lastReleaseBinary
 	copiedCfg.snapCount = 3
-	copiedCfg.baseScheme = "unix" // to avoid port conflict
+
```

Comment:
@xiang90 Sure, then we will need to download separate `v3.0.x` binary until we can drop `v2.7.x`?